samples/managed-app-sample/mainTemplate.json

```diff
@@ -148,7 +148,7 @@
         },
         {
             "type": "Microsoft.ContainerService/managedClusters",
-            "apiVersion": "2022-11-01",
+            "apiVersion": "2024-06-02-preview",
```

Comment:
Lets use GA version 2024-05-01

Reply:
Please add a task to improve the error message related to API version. Also, point to the public doc which helps end user to find the api versions.